modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/configuration/DLSizeLimitConfigurationProviderImpl.java

```diff
@@ -45,6 +47,28 @@ public Map<String, Long> getCompanyMimeTypeSizeLimit(long companyId) {
 			companyId);
 	}
 
+	@Override
+	public String[] getCopyToFailInfo(
+		long companyId, long groupId, long size, Locale locale) {
+
+		long companyMaxSizeToCopy = getCompanyMaxSizeToCopy(companyId);
+		long groupMaxSizeToCopy = getGroupMaxSizeToCopy(groupId);
+
+		if (((size > groupMaxSizeToCopy) && (groupMaxSizeToCopy != 0)) ||
+			((size > companyMaxSizeToCopy) && (companyMaxSizeToCopy != 0))) {
+
+			return new String[] {
+				_language.get(locale, "site-or-instance"),
+				_language.formatStorageSize(groupMaxSizeToCopy, locale)
```

Comment:
This return the _groupMaxSizeToCopy_ always because will be the lesser or is a typo. 
Never will be the companyMaxSizeToCopy?

Reply:
As we show on a call the other day, when we update the companyMaxSizeToCopy, the groupMaxSizeToCopy updates, but we can update groupMaxSizeToCopy without changing companyMaxSizeToCopy, so that's why

Reply:
In my opinion, I think that is a good idea to split this behavior in to check both.
First:
is more accurate, if we change the behavior of the configuration saving, this won't has to be changed.
Second:
we are checking companyMaxSizeToCopy, you are checking it on the if but not using it.

But if you think is not worth to check it, for the behavior you mentioned before, maybe is better idea to remove from the if.